dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -393,8 +393,7 @@ breeze release-management publish-docs \
     --override-versioned
 
 cd "${AIRFLOW_SITE_DIRECTORY}"
-cd post-docs
-python add-back-references.py providers
+breeze release-management add-back-references --airflow-site-directory --gen-type providers
```

Comment:
this doesn't work for me

```
➜  airflow-site git:(main) ✗ breeze release-management add-back-references --airflow-site-directory --gen-type providers
                                                                                                                                                                                     
 Usage: breeze release-management add-back-references [OPTIONS]                                                                                                                      
                                                                                                                                                                                     
                                                                                                                                                                                     
 Try running the '--help' flag for more information.                                                                                                                                 
                                                                                                                                                                                     
╭─ Error ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│ Missing option '-g' / '--gen-type'.                                                                                                                                               │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
                                                                                                                                                                                     
                                                                                                                                                                                     
 To find out more, visit https://github.com/apache/airflow/blob/main/BREEZE.rst                                                                                                      
                                                                                                                                                                                     
    
```

Reply:
Thanks for reporting
I will have a look at it and fix it soon if there's an issue

Reply:
Can you try without the --airflow-directory? What does it complain?

Reply:
Yes `--airflow-site-directory` is a clear problem. It treats `--gen-dir` as value of the `--airflow-site-directory`

Reply:
Yeah, i have fixed this in a follow up PR